routing/cross_mwm_connector.hpp

```diff
@@ -13,25 +14,153 @@
 
 namespace routing
 {
+// @todo(bykoianko) This using should be moved to a special file with TransitId class.
+// Identifier to find and a border edge in neighbouring mwm while cross mwm transition
+// for pedestrian, bicycle and car routing.
+using OsmId = uint64_t;
+
+namespace connector
+{
+uint32_t constexpr kFakeId = std::numeric_limits<uint32_t>::max();
+double constexpr kNoRoute = 0.0;
+
+// This is a type for storing edges weights.
+// Weight is the time required for the route to pass.
+// Weight is measured in seconds rounded upwards.
```

Comment:
Are you sure that it worth here to specify the semantic of Weight?

Reply:
I believe that significant information is written here, but I'm not sure the place is good. The type `Weight` is used mostly in classes `CrossMwmConnector` and in `CrossMwmConnectorSerializer`. Probably it'll be better to move this comment to declaration of member `CrossMwmConnector::m_weights`.